src/app/projects/components/projects/projects-helper.ts

```diff
@@ -1,17 +1,27 @@
 export type ProjectsModals = {
     uploadProject: UploadProjectModal;
+    projectNameSampleProject: ProjectNameSampleProjectModal;
```

Comment:
The name makes sense, but is a bit long in my opinion. What do you think about the name **sampleProjectName**?

Reply:
Yes, I agree (two times project in the name)